### PR TITLE
avoid to create existing node with addr64 0000000000000000

### PR DESCRIPTION
**Pull request by frederic34**

### CODE REVIEW

File: zb-adapter.js [hunk 1]
```diff
@@ -793,6 +793,12 @@ class ZigbeeAdapter extends Adapter {
     for (let i = 0; i < frame.numEntriesThisResponse; i++) {
       const neighborIndex = frame.startIndex + i;
       const neighbor = frame.neighbors[i];
+      if (neighbor.addr64 == '0000000000000000') {
+        const node = this.findNodeByAddr16(neighbor.addr16);
```
Comment (mrstegeman) [inline zb-adapter.js:797]: You're shadowing the outer `node` variable here. Could you use a different name for clarity?

**Comment by frederic34** [comment]
Before:
Sélection_090

After:
Sélection_091

using conbee II

File: zb-adapter.js [hunk 2]
```diff
@@ -191,7 +191,21 @@ class ZigbeeAdapter extends Adapter {
       requestType: 0, // 0 = Single Device Response
       startIndex: 0,
       options: 0,
-    });
+    });    },
```
Comment (mrstegeman) [inline zb-adapter.js:194]: Invalid syntax here.

Comment (frederic34) [reply zb-adapter.js:194]: sorry stange copy paste from my keyboard

**Pull request merged**